synadm/cli/__init__.py

```diff
@@ -376,10 +385,15 @@ def root(ctx, verbose, batch, output, config_file):
     default value 'auto-retrieval' will try to discover the name using the
     method set by --server-discovery."""
 )
+@click.option(
+    "--ssl-verify", "-i",  is_flag=True, default=True, show_default=True,
```

Comment:
`default=True` is odd in this case, as defaults aren't set in any options above but instead below in the `click.prompt` things.

this makes for a UI inconsistency where all other configs use the default value of whatever is set (in `synadm config`) in the options except for `--ssl-verify`.

Reply:
Yes! You are absolutely right, we don't handle defaults that way! Defaults should always be handled via APIHelper.CONFIG and the click.prompt things as you noticed!  Good catch! I fixed this. 

I think now the behaviour is quite fine. Can you please give it another testrun?

To me it looks like this: 
- We silently default to true if the option is missing.
- We display an error when the option is manually set wrongly to some non-boolean string.
  but we don't quit synadm but continue with subsequent errors. This is good, since otherwhise the configurator could not be run. For example:
  
 ```
 (synadm38) ~/git/heurtematte_synadm (master) $ synadm -vv user details testuser2
DEBUG Config entry read. user: admin
DEBUG Config entry read. token: REDACTED
DEBUG Config entry read. base_url: https://peek-a-boo.at
DEBUG Config entry read. admin_path: /_synapse/admin
DEBUG Config entry read. matrix_path: /_matrix
DEBUG Config entry read. timeout: 3600
DEBUG Config entry read. server_discovery: well-known
DEBUG Config entry read. homeserver: peek-a-boo.at
ERROR Config value error: ssl_verify, crap must be boolean
DEBUG Config entry read. ssl_verify: crap
DEBUG Config entry read. format: yaml
DEBUG Formatter in use: yaml - <function dump at 0x7f69b9e70e18>
DEBUG A proper localpart was passed, generating MXID for local homeserver.
INFO  Querying get on https://peek-a-boo.at/_synapse/admin/v2/users/@testuser2:peek-a-boo.at
ERROR OSError while querying Synapse: Could not find a suitable TLS CA certificate bundle, invalid path: crap
User details could not be fetched.
(synadm38) ~/git/heurtematte_synadm (master) $ 
```